### PR TITLE
Nc news5

### PR DESCRIPTION
**Pull request by HarrietHall**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -84,3 +79,26 @@ test("404 : Responds with message -'Not Found' when article id is valid but does
       expect(body.msg).toBe("Not Found");
     });
 });
+
+describe.only("GET /api/articles", () => {
```
Comment (robert112233) [inline __tests__/app.test.js:83]: remove .only!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -84,3 +79,26 @@ test("404 : Responds with message -'Not Found' when article id is valid but does
...
+        expect(article).toHaveLength(13);
+        expect(article).toBeSortedBy("created_at", { descending: true });
+        article.forEach((article) => {
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(String));
```
Comment (robert112233) [inline __tests__/app.test.js:100]: Good to see comment count here

File: db/models/app.model.js [hunk 3]
```diff
@@ -20,3 +18,15 @@ exports.selectArticleById = (article_id) => {
     return rows[0];
   });
 };
+
+exports.selectAllArticles = () => {
+ 
+  const query = "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC "; 
+
```
Comment (robert112233) [inline db/models/app.model.js:25]: Mind-boggling SQL 🥇

**Review by robert112233 — APPROVED** [review]
Good to merge, great work !

File: db/models/app.model.js (hunk 3 above)
Comment (HarrietHall) [reply db/models/app.model.js:25]: nightmare

**Pull request merged**